lidar_prod/tasks/building_validation.py

```diff
@@ -99,7 +99,9 @@ def run(
                 "Preparation : Clustering of candidates buildings & Requesting BDUni"
             )
             if type(input_values) == str:
-                log.info(f"Applying Building Validation to file \n{input_values}")
```

Comment:
Je ne comprends pas pourquoi tu changes 
    log.info(f"Applying Building Validation to file \n{input_values}")
En
    log.info(
        f"Applying Building Validation to file \n{input_values}"
    )
Je trouve cela bien plus compliqué à lire. J'ai vu cette modif à d'autres endroits pour d'autres commit, à chaque fois je trouve que cela rend les choses moins lisbles en plus de rendre le code plus long en hauteur

Reply:
Ce n'est pas vraiment un choix : c'est lié au formatage automatique par `black`. La taille d'une ligne a un maximum qui est appliqué par ce genre de reformatting. Tu peux l'augmenter si tu le souhaites. :)